<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1321</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>371</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="chartLayout1"/>
    </widget>
    <widget class="QPushButton" name="next_Btn">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>620</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>매물보기</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>791</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="chartLayout2"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>340</y>
       <width>791</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="chartLayout3"/>
    </widget>
    <widget class="QComboBox" name="combo_region2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>350</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>광산구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>동구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>북구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>서구</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>그래프</string>
     </property>
    </widget>
    <widget class="QPushButton" name="go_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 연결</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="before_Btn">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>20</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>자료화면</string>
     </property>
    </widget>
    <widget class="QPushButton" name="request_btn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>65</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>매물결정</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>예산 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chat_btn">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>110</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>매물검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>35</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>지역(구)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_money">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>65</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20000</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="stick_btn">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>65</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>그래프</string>
     </property>
    </widget>
    <widget class="QPushButton" name="end_btn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>최종결정</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>327</x>
       <y>35</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>보증금</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>15</y>
       <width>256</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_region">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>65</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>광산구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>북구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>동구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>서구</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>395</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>65</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(단위:만원)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="liner">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>225</y>
       <width>971</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
